frontend/src/types.ts

```diff
@@ -1651,13 +1651,13 @@ export interface Experiment {
     description?: string
     feature_flag_key: string
     // ID of feature flag
-    feature_flag: number
+    feature_flag?: number
```

Comment:
why is this optional now?

Reply:
Because when we create a new experiment there is no feature flag ID attached?